test/client/app/components/higherOrder/hasConfig.test.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { expect } from 'chai';
+import { shallow } from 'enzyme';
+import hasConfig from '../../../../../assets/app/components/higherOrder/hasConfig';
+
+const mock = React.createClass({
```

Comment:
Aha! I'm almost always using stateless components, so I haven't tried this when testing a full-blown React component honestly.